src/System.Runtime/tests/System/Reflection/StrongNameKeyPairTests.cs

```diff
@@ -2,66 +2,101 @@
+        [Theory]
+        [MemberData(nameof(Ctor_ByteArray_TestData))]
+        public void Ctor_ByteArray(byte[] keyPairArray)
+        {
+            // Can't validate anything as this is a nop in .NET Core.
+            // Just make sure it doesn't throw.
+            var keyPair = new StrongNameKeyPair(keyPairArray);
+        }
+
+        [Fact]
+        public void Ctor_NullKeyPairArray_ThrowsArgumentNullException()
+        {
+            AssertExtensions.Throws<ArgumentNullException>("keyPairArray", () => new StrongNameKeyPair((byte[])null));
```

Comment:
Why AssertExtensions instead of Assert?

Reply:
We always use assertextensions for some .NET Native thing

Reply:
Yup, wasn't thinking.